Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -1137,6 +1137,18 @@ - (BOOL)_locked_insetsLayoutMarginsFromSafeArea
 
 @implementation ASDisplayNode (UIViewBridgeAccessibility)
 
+// Walks up the view tree to nil out all the cached accsesibilityElements. This is required when changing
+// accessibility properties like accessibilityViewIsModal.
+// NOTE: If we ship ASExperimentalDoNotCacheAccessibilityElements this is not required.
+- (void)invalidateAccessibilityElements
+{
+  BOOL loaded = _loaded(self);
+  if (loaded && ASDisplayNodeThreadIsMain()) {
```

Comment:
Which is the cheaper / faster check?

- `if (_loaded(self) && ASDisplayNodeThreadIsMain())`
- `if (ASDisplayNodeThreadIsMain() && _loaded(self))`

Reply:
I changed the order because `_loaded` should only be called on the main thread!